diff --git a/libs/core/pyproject.toml b/libs/core/pyproject.toml
index 573bf02..19464c7 100644
--- a/libs/core/pyproject.toml
+++ b/libs/core/pyproject.toml
@@ -97,7 +97,6 @@ build-backend = "poetry.core.masonry.api"
 #
 # https://github.com/tophat/syrupy
 # --snapshot-warn-unused    Prints a warning on unused snapshots rather than fail the test suite.
-addopts = "--snapshot-warn-unused --strict-markers --strict-config --durations=5"
 # Registering custom markers.
 # https://docs.pytest.org/en/7.1.x/example/markers.html#registering-markers
 markers = [
