diff --git a/libs/core/pyproject.toml b/libs/core/pyproject.toml
index 4a26b7f35..74c12d639 100644
--- a/libs/core/pyproject.toml
+++ b/libs/core/pyproject.toml
@@ -91,7 +91,7 @@ ignore = [
 omit = ["tests/*"]
 
 [tool.pytest.ini_options]
-addopts = "--snapshot-warn-unused --strict-markers --strict-config --durations=5"
+addopts = "--strict-markers --strict-config --durations=5"
 markers = [
     "requires: mark tests as requiring a specific library",
     "compile: mark placeholder test used to compile integration tests without running them",
